app/routes/ui/admin_scene.py

```diff
@@ -33,15 +33,17 @@ async def get(self, scene_id):
 
             await self.render(
                 "scene.html",
-                is_last_scene=False,
+                is_last_scene="false",
                 is_admin=True,
                 scene_id=scene_id_int,
                 scenario_name="Edit Scene",
+                scenario_friendly_name="friendly_name",
                 scene_dict=scene_dict,
                 asset_prefix_url=config.get("asset.prefix_url"),
                 navmesh_src=navmesh_src,
                 inspector_url=config.get("inspector_url"),
                 json=json,
+                cur_scene_idx=scene_id_int,
```

Comment:
Why is this field needed?

Reply:
`scene_id` is not the same as `cur_scene_idx`. `scene_id` is the id of the scene in postgres, while `cur_scene_idx` is the index of the current scene.
Example:
`scenario: [5, 77, 32, 23]`, then if someone opens the scenario: `scene_id` would be 5, and `cur_scene_idx` would be 0.

Reply:
On lines 38 and 46, the two fields `scene_id` and `cur_scene_idx` are set to the same thing, so is this a mistake then?

Reply:
Yeah, that wasn't me and was leftover from another PR (https://github.com/uwblueprint/cdc-backend/pull/78) by accident. I can remove it, but it's not a biggie.

Reply:
Removed the unused variable `scene_id`

Reply:
Ah thank you. Makes sense now!